cli/src/pcluster/resources/imagebuilder/parallelcluster_tag.yaml

```diff
@@ -176,7 +187,10 @@ phases:
               NVIDIA_VERSION="$(get_modinfo "nvidia")"
               add_tag "parallelcluster:nvidia_version" "${NVIDIA_VERSION}"
               append_description "nvidia" "${NVIDIA_VERSION}"
-              CUDA_VERSION="$(cat /usr/local/cuda/version.txt | cut -d' ' -f3)"
+              CUDA_VERSION="$(cat /usr/local/cuda/version.json | jq -r '.cuda.version')"
+              if [ -z "${CUDA_VERSION}" ]; then
+                CUDA_VERSION=$(get_cookbook_attribute "default['cluster']['nvidia']['cuda_version']" | tr -d '\n' | cut -d = -f 2 | xargs)
+              fi
```

Comment:
The version in version.json includes (what I assume is) a build date, so something like 11.4.20220113 whereas the version in the cookbook is just the version (e.g. 11.4) - I'm not sure having different ways of tagging the version on different architectures is the right answer.